probe/appclient/multi_client.go

```diff
@@ -100,9 +102,7 @@ func (c *multiClient) Set(hostname string, endpoints []string) {
 	hostIDs := report.MakeIDList()
 	for tuple := range clients {
 		hostIDs = hostIDs.Add(tuple.ID)
-
-		_, ok := c.clients[tuple.ID]
-		if !ok {
```

Comment:
Actually, I am wrong, it's also used by [`Publish()`](https://github.com/weaveworks/scope/blob/74e508671ce26156118f9b4f60aaec72c7de2303/probe/appclient/multi_client.go#L172). Fixing ...